Content.Server/GameObjects/Components/Damage/RuinableComponent.cs

```diff
@@ -43,7 +47,7 @@ public override void ExposeData(ObjectSerializer serializer)
                 },
                 () => Thresholds.TryGetValue(DamageState.Dead, out var value) ? value : (int?) null);
 
-            serializer.DataField(this, ruinable => ruinable.DestroySound, "destroySound", string.Empty);
+            serializer.DataField(this, ruinable => ruinable.DestroySounds, "destroySounds", new List<string>());
```

Comment:
Make it specify either a single sound, or a soundcollection. Have the sound collection always have a higher priority.